### PR TITLE
Add function &Rest.Order.get/2

### PR DESCRIPTION
**Pull request by ihorkatkov**
By having this function, a user can retrieve his orders from the
exchange

### CODE REVIEW

File: lib/ex_bitmex/rest/orders.ex [hunk 1]
```diff
@@ -67,6 +67,20 @@ defmodule ExBitmex.Rest.Orders do
     |> parse_response
   end
 
+  @spec get(credentials, params) ::
+          {:ok, order, rate_limit} | {:error, create_error_reason, rate_limit | nil}
```
Comment (kimquy) [inline lib/ex_bitmex/rest/orders.ex]: Wondering that if this get function return just an `order` or `list of orders` ?

Comment (kimquy) [reply lib/ex_bitmex/rest/orders.ex]: Later on we can we can add dialyzer to CI to fix this issue I think

Comment (ihorkatkov) [reply lib/ex_bitmex/rest/orders.ex]: nice catch. It always will return a list

**Comment by kimquy** [comment]
LGTM

**Pull request merged**